### DISCUSSION TITLE
How can I remove a useless service

### DISCUSSION THREAD

**Discussion by JianminWu** [apache/skywalking#10756] [Q&A] [answered]
When I try skywalking. I have not write a groupName. Next , I added a groupName by {groupName}::{serviceName}. But in dashboard,there are two same service. one has a groupName ,other has not. I want to remove that service without groupName.
I use skywalking 9.3.0-7ad0406 dashboard. I choose mysql to storage the data.
please help! thans a lot!!
![image](https://user-images.githubusercontent.com/31835440/236105457-fc761dfd-e9d9-4475-87c8-4bb4a6f85331.png)

**Comment by wu-sheng** [comment, marked as answer]
You can't do that manually. TTL(metric config) would do that automatically, the default is 7 days.

**Reply by JianminWu** [reply]
Can I change this config? such as 1 days? should I restart the skywalking?

**Reply by wu-sheng** [reply]
then all metrics would be gone in 1 day(TTL = 2). You could read the docs of TTL.
